deploy/openj9-chart/README.md

```diff
@@ -0,0 +1,106 @@
+# OpenJ9 Helm Chart
+
+## Introduction
+
+Eclipse OpenJ9 is an independent implementation of a Java Virtual Machine.
+
+The OpenJ9 Chart allows you to deploy and manage java applications running on OpenJ9 into OKD or OpenShift clusters. You can also perform Day-2 operations such as rolling update your application, enabling JITServer technology for continued performance enhancements or gathering JVM dumps using the operator.
+
+## Resources Required
+
+### System
+
+* CPU Requested : 1 CPU
+* Memory Requested : 256 MB
+
+### Storage
+
+* Currently no storage or volume needed.
+
+## Chart Details
+
+* Install one `deployment` running Adopt OpenJ9 image
+* Install one `service` that is JITServer technology enabled by default
+
+## Prerequisites
+
+### Red Hat OpenShift SecurityContextConstraints Requirements
+
+This chart does not require SecurityContextConstraints to be bound to the target namespace prior to installation.
+
+### PodSecurityPolicy Requirements
+
+This chart does not require PodSecurityPolicy to be bound to the target namespace prior to installation.
+
+### PodDisruptionBudgets Requirement
+
+This chart does not require PodDisruptionBudgets to be bound to the target namespace prior to installation.
+
+## Installing the Chart
+
+``` bash
+helm install my-release PATH_TO_OPENJ9_CHART
```

Comment:
Is the `my-release` name part of the helm command?  Could it be replaced with something more specific?

Reply:
`my-release` is just a place holder, a name for the installed helm chart, which is also called a helm release. Should I change the name to `example-openj9` instead? Or please suggest a more intuitive name, thanks.